public/static/docs/command-reference/checkout.md

```diff
@@ -16,60 +16,49 @@ positional arguments:
 
 ## Description
 
-[DVC-files](/doc/user-guide/dvc-file-format) in a <abbr>project</abbr> specify
-which instance of each data file or directory should be used, with the checksums
-saved in the `outs` field. The `dvc checkout` command updates the workspace data
-to match with the <abbr>cached</abbr> files that correspond to those checksums.
-
-Using an SCM like Git, the DVC-files are kept under version control. At a given
-branch or tag of the SCM repository, the DVC-files will contain checksums for
-the corresponding data files kept in the cache. After an SCM command like
-`git checkout` is run, the DVC-files will change to the state at the specified
-branch or commit or tag. Afterwards, the `dvc checkout` command is required in
-order to synchronize the data files with the currently checked out DVC-files.
-
-This command must be executed after `git checkout` since Git doesn't track files
-that are under DVC control. For convenience a Git hook is available, simply by
-running `dvc install`, that will automate running `dvc checkout` after
-`git checkout`. See `dvc install` for more information.
-
-The execution of `dvc checkout` does:
-
-- Scan the `outs` entries in DVC-files to compare with the currently checked out
-  data files. The scanned DVC-files is limited by the listed `targets` (if any)
-  on the command line. And if the `--with-deps` option is specified, it scans
-  backward from the given `targets` in the corresponding
-  [pipeline](/doc/command-reference/pipeline).
-
-- For any data files where the checksum doesn't match their DVC-file entry, the
-  data file is restored from the cache. The link strategy used (`reflink`,
-  `hardlink`, `symlink`, or `copy`) depends on the OS and the configured value
-  for `cache.type` – See `dvc config cache`.
-
-Note that this command by default tries NOT to copy files between the cache and
-the workspace, using reflinks instead when supported by the file system. (Refer
-to
+[DVC-files](/doc/user-guide/dvc-file-format) are essentially placeholders that
```

Comment:
can you actually use "placeholders that point ..." ... or should it be just "placeholders for smth."?

Reply:
I would also remove `essentially`

Reply:
also, it better to say something like "specific version of data ..."

Reply:
I think both are correct. They are placeholders for something, and they point to that something. Do you think it's redundant? I'm just trying to be extra clear about how DVC-files work.

Reply:
p.s. removed "essentially" and changed "the actual data"->"specific version of data"

Reply:
> sorry, still feels strange to me :)
> it's not right to mix placeholder and pointer, probably

OK, changed per your suggestion to:

`DVC-files act as pointers to specific version of data files or directories under DVC control.`

But omitted the part about `outs` field because it's explain further down in the implementation details, per https://github.com/iterative/dvc.org/pull/864#pullrequestreview-338470052.

Good?

Reply:
👍